Javascript Lint pipeline

mateiw opened the pull request:
JavaScript pipeline to run ESLint and Flow on PRs.

MalloZup commented on jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint:
```diff
@@ -0,0 +1,51 @@
+   }
+   // run only on specific hosts
+    agent { label 'suse-manager-unit-tests' }
+   // trigger
```
how do we schedule this pipeline if not auto-triggered?

MalloZup commented on jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint:
```diff
@@ -0,0 +1,51 @@
+                " -f \".js\" " +
+                " -t susemanager-utils/testing/automation/javascript-lint.sh" +
+                " -g ${env.WORKSPACE} " +
+                "--check " // --changed_since 3600 
```
if we comment out this we will reach the Rate limit of github ( `changed_since`)

MalloZup commented on jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint:
```diff
@@ -0,0 +1,51 @@
+    jslint_check = "gitarro.ruby2.1  -r SUSE/spacewalk" + 
+                " -c javascript_lint -d \"javascript-lint\" " +
+                " -f \".js\" " +
+                " -t susemanager-utils/testing/automation/javascript-lint.sh" +
```
`nitpick`: we can have this script inside of the pipeline, But is also ok in manager

mateiw commented:
@MalloZup I addressed your comments apart from `javascript-lint.sh`. I would keep it inside manager for consistency with the other scripts like `java-checkstyle.sh`, etc.
